addons/mail/migrations/12.0.1.0/post-migration.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+from psycopg2.extensions import AsIs
+
+
+def fill_mail_blacklist_res_partner(cr):
+    cr.execute(
+        """
+        INSERT INTO mail_blacklist (email, active,
+            create_uid, create_date, write_uid, write_date)
+        SELECT email, active, create_uid, create_date, write_uid, write_date
+        FROM res_partner rp
+        WHERE rp.%s
```

Comment:
From @rvalyi error comment:
```suggestion
        WHERE rp.%s AND rp.email IS NOT NULL
```